Network/Mail/SMTP.hs

```diff
@@ -309,27 +318,19 @@ simpleMail from to cc bcc subject parts =
 
 -- | Construct a plain text 'Part'
 plainTextPart :: TL.Text -> Part
-plainTextPart = Part "text/plain; charset=utf-8"
-              QuotedPrintableText Nothing [] . TL.encodeUtf8
+plainTextPart body = Part "text/plain; charset=utf-8"
```

Comment:
@parsonsmatt  `mime-mail` has `plainPart`, `htmlPart`, and `filePart`. Should I just remove these and prefer ones from `mime-mail` instead?